modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -218,6 +180,49 @@ enum ImwriteHDRCompressionFlags {
 
 //! @} imgcodecs_flags
 
+/** @brief Represents an animation with multiple frames.
+The Animation struct is used to store and manage the data for an animated sequence.
+It includes information about the number of times the animation should loop, the background color,
+timestamps for each frame, and the frames themselves.
+*/
+struct CV_EXPORTS_W_SIMPLE Animation
+{
+    //! Number of times the animation should loop. 0 means infinite looping.
+    CV_PROP_RW int loop_count;
+    //! Background color of the animation in RGBA format.
+    CV_PROP_RW int bgcolor;
+    //! Timestamps for each frame in milliseconds.
+    CV_PROP_RW std::vector<int> timestamps;
+    //! Vector of frames, where each Mat represents a single frame.
+    CV_PROP_RW std::vector<Mat> frames;
+    //! Quality of the animation, typically between 0 (lowest) and 100 (highest).
+    CV_PROP_RW int quality;
```

Comment:
That should be an extra parameter in imwriteanimation because it is not filled by imreadanimation

Reply:
i will remove it. IMWRITE_WEBP_QUALITY parameter does the job.